snap/README.md

```diff
@@ -0,0 +1,43 @@
+# Kata Containers Snap
+
+This directory contains the resources needed to build the Kata Containers
+[snap][1] image.
+
+## Initial setup
+
+*Ubuntu Bionic*
+
+```
+$ sudo apt-get install -y snap
+$ git clone https://github.com/kata-containers/packaging
+```
+
+## Build snap image
+
+Run next command at the root directory of the packaging repository.
+
+```
+$ snapcraft -d
+```
+
+## Install snap (developer)
+
+```
+$ sudo snap install --classic --dangerous kata-containers_[VERSION]_[ARCH].snap
+```
+
+Replace `VERSION` with the current version of Kata Containers and `ARCH` with
+the system architecture.
+
+## Configuring Kata Containers ##
+
+
+## Snap vs Packages ##
+
+
+## Limitations
+
+The miniOS image is not included in the snap image due to is not possible to
```

Comment:
- Could you add a link to osbuilder and/or https://github.com/kata-containers/documentation/blob/master/architecture.md#root-filesystem-image.

- I'd reword this sentence with something like:
  > The miniOS image is not included in the snap image as it is not possible for QEMU to open a guest RAM backing store on a read-only filesystem.

Reply:
will do, thanks